src/DI/ContainerBuilder.php

```diff
@@ -327,7 +331,8 @@ public function prepareClassList()
 		foreach ($this->definitions as $name => $def) {
 			if ($class = $def->getImplement() ?: $def->getClass()) {
 				foreach (class_parents($class) + class_implements($class) + [$class] as $parent) {
-					$this->classes[$parent][$def->isAutowired() && empty($excludedClasses[$parent])][] = (string) $name;
+					$autowired = isset($excludedClasses[$parent]) ? NULL : $def->isAutowired();
```

Comment:
This breaks ApplicationExtension which looks for presenters by excluded class name: https://api.nette.org/2.3.5/source-Bridges.ApplicationDI.ApplicationExtension.php.html#97